html-css/week1/styles.css

```diff
@@ -0,0 +1,246 @@
+:root {
+    --main-red: #be3144;
+    --main-gray: #303841;
+    --main-lightgray: #0d2746;
+    --main-white: #f0f0f0;
+  }
+  * {
+    margin: 0;
+  }
+  
+  body {
+    font-family: sans-serif;
+    font-weight: 400;
+    line-height: 1.4;
+    color: var(--main-white);
+  }
+  .nav-container {
+    background-image: linear-gradient(62deg, #3a3d40 0%, #050505 100%);
+  }
+  
+  .navbar {
+    position:relative;
+    top: 20px;
+    left: 1em; 
+    border-radius: 14px;
+    justify-content: inherit;
+    font-size: 18px;
+    width:97%;
+    background: var(--main-red);  
+  }
+
+  .nav-list{
+    display: flex;
+    li {
+      list-style-type: none;
+    }
+  }
+
+  .nav-list li > a {
+    color: inherit;
+    text-decoration: none;
+    padding: 10px;
```

Comment:
I noticed you've added `padding `to `.nav-list li > a`, and while this works, I’d recommend considering an alternative approach: using margin on `.nav-list li` instead